features/zoek-met-bsn.feature

```diff
@@ -0,0 +1,101 @@
+        | naam                | waarde    |
+        | burgerservicenummer | 999991802 |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                        |
+        | type                | ZoekMetBurgerservicenummer    |
+        | burgerservicenummer | 999999321,999995492,999991802 |
+        | fields              | burgerservicenummer           |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999999321           |
+        | 999995492           |
+        | 999991802           |
+
+    @fout-case
+    Scenario: Zoek zonder burgerservicenummers
+        Als personen wordt gezocht met de volgende parameters
+        | naam   | waarde                     |
+        | type   | ZoekMetBurgerservicenummer |
+        | fields | burgerservicenummer        |
+        Dan bevat de response de volgende gegevens
+        | naam     | waarde                                              |
+        | title    | Een of meerdere parameters zijn niet correct.       |
+        | status   | 400                                                 |
+        | detail   | De foutieve parameter(s) zijn: burgerservicenummer. |
+        | code     | paramsValidation                                    |
+        | instance | /personen                                           |
+        En bevat de response een invalidParams met de volgende gegevens
+        | code     | name                | reason                  |
+        | required | burgerservicenummer | Parameter is verplicht. |
+
+    @fout-case
+    Abstract Scenario: <titel>
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | <burgerservicenummers>     |
+        | fields              | burgerservicenummer        |
+        Dan bevat de response de volgende gegevens
+        | naam     | waarde                                        |
+        | title    | Één of meerdere parameters zijn niet correct. |
+        | status   | 400                                           |
+        | detail   | TODO                                          |
+        | code     | TODO                                          |
+        | instance | TODO                                          |
+        En bevat de response een invalidParams met de volgende gegevens
+        | code     | name                | reason                  |
+        | required | burgerservicenummer | Parameter is verplicht. |
+
+        Voorbeelden:
+        | burgerservicenummers | titel                                   |
+        |                      | Zoek met leeg burgerservicenummer lijst |
+        | 1234567890           | Zoek met ongeldig burgerservicenummer   |
+
+Rule: Er mag maximaal 10 burgerservicenummers worden opgegeven
```

Comment:
Waar komt deze regel vandaan? In v1.3 is er geen maximum gespecificeerd. Het zoeken heeft dan in het algemeen een maximum, maar die kan hoger zijn dan 10

Reply:
Kun je aangeven wat het maximum is? Het is volgens mij gebruiksvriendelijker om dat dan als maximum op te geven, zodat consumers daar rekening mee kunnen houden

Reply:
besloten dat maximum 20 wordt